### DISCUSSION TITLE
How do I insert a blank line between two tables?

### DISCUSSION THREAD

**Discussion by JoBurgard** [outline/outline#6765] [General] [answered]
Hello,
I am not able to put a blank line between two tables. I searched in the docs and the discussions.

Am I missing some hotkey? I tried ctrl+enter but that doesn't work and it exits the edit mode, too.

![image](https://github.com/outline/outline/assets/4430298/dddf673f-fa01-4edd-9719-9d711b9e522e)

**Comment by tommoor** [comment, marked as answer]
Tricky indeed! This somewhat relates to https://github.com/outline/outline/issues/6281 where we need to change the behavior of enter, `Mod+Enter` could insert a new line after the table in this case.

The best workaround right now would be to CMD+Click on a paragraph to select it and then drag the paragraph between the two tables.
